spec/models/like_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.new(
+    name: 'Jim', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+    bio: 'Teacher from Mexico',
+    posts_counter: 0
+  )
+  post = Post.new(
+    author: user, title: 'Hola', text: 'This is my first post',
+    likes_counter: 0, comments_counter: 0
+  )
+
+  subject do
+    Like.new(post:, author: user)
```

Comment:
- [ ] It would be better if we fix these errors I faced while running the rspec test. The screenshot will help us debug the code keeping an eye on file names and line number
![Screenshot from 2022-03-31 23-25-29](https://user-images.githubusercontent.com/46388113/161165918-be4b1df5-db75-4dc9-8f1a-8779bf1a1621.png)

Reply:
Hello @addod19,

The tests are running just fine on my end.
![image](https://user-images.githubusercontent.com/62290545/161167310-a73d74aa-c775-444e-b3c9-32589fd9cce0.png)